compiler/rustc_target/src/spec/abi/mod.rs

```diff
@@ -79,6 +79,7 @@ impl Abi {
         match self {
             Self::C { .. }
             | Self::Cdecl { .. }
+            | Self::System { .. }
```

Comment:
Please also update the comment above, it is now contradicting the code.

Cc @Soveu @jackh726 who added this comment in https://github.com/rust-lang/rust/pull/97971
Cc @bjorn3 for another ABI expert